File: JAVA/Day02/StackExample.java
import java.util.*;

public class StackExample {
    public static void main(String[] args) {
        Stack<Integer> stack = new Stack<>();
        
        stack.push(10);
        stack.push(20);
        stack.push(30);
        
        System.out.println("Stack: " + stack);
        
        int poppedElement = stack.pop();
        System.out.println("Popped Element: " + poppedElement);
        
        int topElement = stack.peek();
        System.out.println("Top Element: " + topElement);
        
        boolean isEmpty = stack.isEmpty();
        System.out.println("Is Stack Empty? " + isEmpty);
    }
}

File: C#/Day02/StackExample.cs
using System;
using System.Collections.Generic;

public class StackExample {
    public static void Main(string[] args) {
        Stack<int> stack = new Stack<int>();
        
        stack.Push(10);
        stack.Push(20);
        stack.Push(30);
        
        Console.WriteLine("Stack: " + string.Join(", ", stack));
        
        int poppedElement = stack.Pop();
        Console.WriteLine("Popped Element: " + poppedElement);
        
        int topElement = stack.Peek();
        Console.WriteLine("Top Element: " + topElement);
        
        bool isEmpty = stack.Count == 0;
        Console.WriteLine("Is Stack Empty? " + isEmpty);
    }
}
